latch_cli/services/init/example_snakemake/Snakefile

```diff
@@ -50,18 +50,18 @@ rule bcftools_call:
     input:
         fa="genome/genome.fa",
         bam=expand("sorted_reads/{sample}.bam", sample=SAMPLES),
-        bai=expand("sorted_reads/{sample}.bam.bai", sample=SAMPLES)
+        bai=expand("sorted_reads/{sample}.bam.bai", sample=SAMPLES),
     output:
-        "calls/all.vcf"
+        "calls/all.vcf",
     shell:
         "bcftools mpileup -f {input.fa} {input.bam} | "
         "bcftools call -mv - > {output}"
 
 
 rule plot_quals:
     input:
-        "calls/all.vcf"
+        "calls/all.vcf",
```

Comment:
uhhh i thought trailing commas are an error in snakemake? at least on some versions?